pkg/client/remotecommand/remotecommand_test.go

```diff
@@ -112,8 +126,41 @@ func (s *fakeUpgradeStream) Read(p []byte) (int, error) {
 	defer s.lock.Unlock()
 	s.readCalled = true
 	b := []byte(s.data)
-	n := copy(p, b)
-	return n, io.EOF
+
+	// resume reading where we left off
+	start := s.index
+
+	// default to 10 bytes per invocation
```

Comment:
Is this really a good analog of the "real" upgrade stream object? I suspect not?